templates/.goreleaser.yml.tpl

```
@@ -12,28 +12,29 @@ builds:
 {{- $cmdName = (index (keys $cmdName) 0) }}
 {{- $opts = (index . $cmdName | default (dict)) }}
 {{- end }}
-- main: ./cmd/{{ $cmdName }}
-  id: &name {{ $cmdName }}
-  binary: *name
-  goos:
-  - linux
-  - darwin
-  goarch:
-  - amd64
-  - arm64
-  ldflags:
-   - '-w -s -X "github.com/getoutreach/gobox/pkg/app.Version=v{{ "{{" }} .Version {{ "}}" }}"'
-   {{- if not $opts.delibird }}
-   - '-X "main.HoneycombTracingKey={{ "{{" }} .Env.HONEYCOMB_APIKEY {{ "}}" }}"'
-   - '-X "main.TeleforkAPIKey={{ "{{" }} .Env.TELEFORK_APIKEY {{ "}}" }}"'
-   {{- end }}
-  env:
-  - CGO_ENABLED={{ stencil.ApplyTemplate "cgoEnabled" | trim }}
-  {{- $blockName := (printf "%vAdditionalEnv" $cmdName) }}
-  ### <<Stencil::Block({{ $blockName }})>>
-{{ (file.Block $blockName) | trim | indent 2 }}
-  ### <</Stencil::Block>>
-  {{- end }}
+  - main: ./cmd/{{ $cmdName }}
+    id: &name {{ $cmdName }}
+    binary: *name
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    ldflags:
+      - '-w -s -X "github.com/getoutreach/gobox/pkg/app.Version=v{{ "{{" }} .Version {{ "}}" }}"'
+      {{- if not $opts.delibird }}
+      - '-X "main.HoneycombTracingKey={{ "{{" }} .Env.HONEYCOMB_APIKEY {{ "}}" }}"'
+      - '-X "main.TeleforkAPIKey={{ "{{" }} .Env.TELEFORK_APIKEY {{ "}}" }}"'
+      {{- end }}
+    env:
+      - CGO_ENABLED={{ stencil.ApplyTemplate "cgoEnabled" | trim }}
+      {{- $cmd :=  $cmdName | split "-" }}
+      {{- $blockName := (printf "%vAdditionalEnv" $cmd._0) }}
```

suggestion: this will be problematic if, for example, we made a repo with a bunch of `kubectl` plugins (they are named `kubectl-foo` and `kubectl-bar`). Do underscores work? If so, you can do this:
```suggestion
      {{- $blockName := (printf "%vAdditionalEnv" ($cmdName | replace "-" "_")) }}
```

---

If underscores don't work, I'd just get rid of it altogether:
```suggestion
      {{- $blockName := (printf "%vAdditionalEnv" ($cmdName | replace "-" "")) }}
```

---

I see, good point. I will try if underscore works in block.

---

Underscore does not work in block, replaced "-" with "".